providers/component-protocol/components/list/model.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package list
+
+import "github.com/erda-project/erda-infra/providers/component-protocol/cptype"
+
+type (
+	// Data includes list of items.
+	Data struct {
+		List       []Item                                   `json:"list,omitempty"`
+		PageNo     uint64                                   `json:"pageNo,omitempty"`
+		PageSize   uint64                                   `json:"pageSize,omitempty"`
+		Total      uint64                                   `json:"total,omitempty"`
+		Operations map[cptype.OperationKey]cptype.Operation `json:"operations,omitempty"`
+		UserIDs    []string                                 `json:"userIDs,omitempty"`
+	}
+
+	// Item minimum unit of list
+	Item struct {
+		// uniq id of the item, e.g: appID, projID, ...
+		ID    string `json:"id,omitempty"`
+		Title string `json:"title,omitempty"`
+		// logo link of title
+		Logo          string      `json:"logo,omitempty"`
+		Star          bool        `json:"star,omitempty"`
+		Labels        []ItemLabel `json:"labels,omitempty"`
+		Description   string      `json:"description,omitempty"`
+		BackgroundImg string      `json:"backgroundImg,omitempty"`
```

Comment:
Img means URL or any others, is Ambiguous.

Reply:
yes, it's a url link

Reply:
// OpItemBasicServerData .
type OpItemBasicServerData struct {
	// open a new fronted page
	JumpOut bool `json:"jumpOut,omitempty"`
	// the jump out target of the new opened fronted page, e.g: projectAllIssue page
	Target string `json:"target,omitempty"`
	// params needed for jumping to the new page, e.g: projectId
	Params map[string]interface{} `json:"params,omitempty"`
	// the query condition of the new page, e.g: issueFilter__urlQuery
	Query map[string]interface{} `json:"query,omitempty"`
}

Reply:
done